test_case_gen.py

```diff
@@ -0,0 +1,249 @@
+"""
+Usage:
+    python2 test_case_gen.py
+
+Generates Topologies for Test Cases 0, 1, 2 each using three different
+congestion control algorithms (Dummy (always window size 1), Reno, FAST) and
+saves them to json.
+
+json's are stored in data/test_case_<number>_<flow type>.json
+"""
+from network_simulator.network_topology import *
+from network_simulator.host import *
+from network_simulator.common import *
+from network_simulator.router import *
+from network_simulator.flow import *
+from network_simulator.link import *
+
+if __name__ == "__main__":
+    # test case 0
+    links = list()
+    l1 = Link(end_1_addr = "H1", end_2_addr = "H2", link_buffer =
+            LinkBuffer(max_buffer_size_bits = 64 * KILOBYTE),
+            static_delay_sec = 0.01, capacity_bps = 10 * MEGABIT)
+    links.append(l1)
+
+    hosts = list()
+    h1 = Host("H1")
+    h2 = Host("H2")
+    hosts.append(h1)
+    hosts.append(h2)
+
+    # generate for FlowDummy
+    flows = list()
+    f1 = FlowDummy(flow_id = "F1", source_addr = "H1", dest_addr = "H2",

```

Comment:
When using named parameters, don't have spaces around "=" signs. PyCharm should warn about this.